types/queries/all_documents.sql

```diff
@@ -20,4 +21,7 @@ from document
   left join
     contributor_attribution on contributor_attribution.document_id = document.id
   left join contributor on contributor.id = contributor_attribution.contributor_id
-group by document.id
+  left join user_bookmarked_document as ubd on ubd.document_id = document.id
```

Comment:
I think this should be removed now that we are fetching the `bookmarked_on` field lazily, right??

FWIW, if you were going to fetch `bookmarked_on` for many documents, you'll have to also pass in the `id` of the signed-in user and add a condition to the where clause. Ie. something like `AND ubd.user_id = $whatever`. Otherwise this query might start doing weird things once multiple users bookmark the same document.